platform/viewer/package.json

```diff
@@ -27,8 +27,14 @@
     "start": "yarn run dev",
     "orthanc:up": "docker-compose -f docker/Nginx-Orthanc/docker-compose.yml up",
     "test:e2e": "cypress open",
+    "test:e2e:ci": "start-server-and-test start http://localhost:3000 cy:run:ci",
+    "make-script-tag-example": "rm -rf script-tag-test && mkdir script-tag-test && cp dist/index.umd.* script-tag-test && cp -R public/* script-tag-test && cp -R cypress/support/script-tag/* script-tag-test",
```

Comment:
Can we have `webpack-html-plugin` generate a good entry point when we build the script tag's output? Webpack should be cleaning the output directory already, and it's where the `index.umd.js` will live. Let's just include an alternative template?